ci-services/jenkins.js

```diff
@@ -3,10 +3,18 @@ const env = process.env
 const _ = require('lodash')
 const gitHelpers = require('../lib/git-helpers')
 
+// Jenkins reports the branch name and Git URL in a couple of different places depending on use of the new
+// pipeline vs. older job types.
+const branchName = env.CHANGE_BRANCH || env.GIT_BRANCH
+const gitUrl = env.CHANGE_URL || env.GIT_URL
+
+// Build number is available in Jenkins pipeline builds; for others, defer to asking GIT.
+const buildNumber = env.BUILD_NUMBER || gitHelpers.getNumberOfCommitsOnBranch(branchName)
+
 module.exports = {
-  gitUrl: env.GIT_URL,
-  branchName: _.drop(_.split(env.GIT_BRANCH, '/')).join('/'),
```

Comment:
The `branchName` mangling code seems to not have survived this. Any particular reason?